cf-agent/verify_exec.c

```diff
@@ -292,21 +291,17 @@ static ActionResult RepairExec(EvalContext *ctx, const Attributes *a,
 
     CommandPrefix(cmdline, comm);
 
+    bool do_work_here = true;
+
     if (a->transaction.background)
     {
-#ifdef __MINGW32__
-        do_work_here = true;
-#else
         Log(LOG_LEVEL_VERBOSE, "Backgrounding job '%s'", cmdline);
+#ifndef __MINGW32__
```

Comment:
The log message should not show up on windows where backgrounding doesn't happen, right?

Reply:
Good point, this is just the cherry-pick, though.

Reply:
https://github.com/cfengine/core/pull/4259